### PR TITLE
Fix some system service imports going missing after reload

### PR DESCRIPTION
**Pull request by wallyqs**
On reload some of the imports from the system account where going missing on reload, this adds them back after a reload:

```
$SYS.REQ.SERVER.PING.CONNZ
$SYS.REQ.ACCOUNT.PING.STATZ
$SYS.REQ.ACCOUNT.PING.CONNZ
```

### CODE REVIEW

**Review by neilalexander — APPROVED** [review]
LGTM

File: server/reload_test.go [hunk 1]
```diff
@@ -2603,6 +2603,23 @@ func TestConfigReloadAccountUsers(t *testing.T) {
 		t.Fatalf("Error on subscribe: %v", err)
 	}
 
+	// confirm subscriptions before and after reload.
+	var expectedSubs uint32 = 4
+	sAcc, _ := s.LookupAccount("synadia")
+	sAcc.mu.RLock()
```
Comment (derekcollison) [inline server/reload_test.go:2609]: Capture err and do require_NoError(t, err)

Comment (wallyqs) [reply server/reload_test.go:2610]: thanks, updated

**Review by derekcollison — COMMENTED** [review]
LGTM

**Review by derekcollison — APPROVED** [review]
LGTM

**Pull request merged**